doctrine/doctrine-module/2013-05-16.yaml

```diff
@@ -2,10 +2,10 @@ title:     Authentication Vulnerability - possible attempt to login via zero-val
 link:      https://github.com/doctrine/DoctrineModule/issues/249
 cve:       ~
 branches:
-    master:
-        time:     2013-05-16 18:29:36
-        versions: [<=0.7.1]
     0.7.x-dev:
         time:     2013-05-16 18:27:30
         versions: [<=0.7.1]
+    master:
+        time:     2013-05-16 18:29:36
+        versions: [>=0.7.0,<=0.7.1] # the master branch was fixed separately must was not yet released
```

Comment:
well, it depends how the SensioLabs security checker handles the validation of affected dev versions. The 0.7.x and master branches have been fixed at different times.

Changing the constraint in this one was actually necessary just because I was lazy about making the validator more complex (as the max bound is the same in both branches, it may be accepted by a more advanced validation implementation)